api/src/main/java/org/openmrs/module/m2sysbiometrics/client/M2SysV105Client.java

```diff
@@ -80,10 +79,17 @@ public EnrollmentResult enroll(BiometricSubject localSubject) {
             enrollmentStatus = EnrollmentStatus.ALREADY_REGISTERED;
         }
 
-        if (nationalBioServerClient.isServerUrlConfigured() && !fingerScanStatus.isRegisteredNationally()) {
-            registrationService.registerNationally(capture);
-            fingerScanStatus = searchService.checkIfFingerScanExists(capture);
-            nationalSubject = new BiometricSubject(fingerScanStatus.getNationalBiometricSubject().getSubjectId());
+        BiometricSubject nationalSubject = new BiometricSubject("");
+        if (fingerScanStatus.isRegisteredNationally()) {
+            nationalSubject = fingerScanStatus.getNationalBiometricSubject();
+        } else {
```

Comment:
Maybe try 'else if' instead of 'else' and 'if' since there is no line which executes for 'else' statement

Reply:
You're right